src/Forms/Main.cs

```diff
@@ -12175,15 +12175,29 @@ private bool ImportSubtitleFromTransportStream(string fileName)
             ShowStatus(string.Empty);
             TaskbarList.SetProgressState(Handle, TaskbarButtonProgressFlags.NoProgress);
 
-            if (tsParser.SubtitlePacketIds.Count == 0)
+            if (tsParser.SubtitlePacketIds.Count == 0 && tsParser.TeletextSubtitlesLookup.Count == 0)
             {
                 MessageBox.Show(_language.NoSubtitlesFound);
                 _exitWhenLoaded = _loading;
                 return false;
             }
 
-            int packedId = tsParser.SubtitlePacketIds[0];
-            if (tsParser.SubtitlePacketIds.Count > 1)
+            if (tsParser.SubtitlePacketIds.Count == 0 & tsParser.TeletextSubtitlesLookup.Count == 1 && tsParser.TeletextSubtitlesLookup.First().Value.Count() == 1)
+            {

```

Comment:
@niksedk
typo: **`&`** ==> **`&&`**.
```
tsParser.SubtitlePacketIds.Count == 0 && tsParser.TeletextSubtitlesLookup.Count == 1
```

Reply:
Fixed in https://github.com/SubtitleEdit/subtitleedit/commit/cf36eb9e69328b88818c6249351779075d0a73ad